packages/paste-core/primitives/box/package.json

```diff
@@ -29,6 +29,7 @@
   },
   "peerDependencies": {
     "@emotion/styled": "^10.0.10",
+    "@styled-system/css": "^5.1.5",
```

Comment:
You might want to run `npx @yarnpkg/doctor`. My hunch is that you're going to need to add this as a peer dep to every package that uses box, and also add it as a dev dep to any package that has a peer dep of box.